includes/class-admin.php

```diff
@@ -668,6 +679,27 @@ public static function manage_comments_custom_column( $column, $comment_id ) {
 		}
 	}
 
+	/**
+	 * Add the new ActivityPub comment types to the comment types dropdown.
+	 *
+	 * @param array $types The existing comment types.
+	 *
+	 * @return array The extended comment types.
+	 */
+	public static function comment_types_dropdown( $types ) {
+		global $activitypub_comment_types;
```

Comment:
Same as above, should we use `Comment::get_comment_types()` instead of the global?